libs/database-entities/src/main/java/gov/cdc/nbs/repository/PersonRepository.java

```diff
@@ -7,14 +7,18 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.querydsl.QuerydslPredicateExecutor;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 import gov.cdc.nbs.entity.odse.Person;
 
 @Repository
 public interface PersonRepository extends JpaRepository<Person, Long>, QuerydslPredicateExecutor<Person> {
-        @Query("SELECT coalesce(max(p.id), 0) FROM Person p")
-        Long getMaxId();
+	@Query("SELECT coalesce(max(p.id), 0) FROM Person p")
+	Long getMaxId();
 
-        Page<Person> findByIdIn(List<Long> ids, Pageable pageable);
+	Page<Person> findByIdIn(List<Long> ids, Pageable pageable);
+
+	@Query(value="SELECT p.person_uid from Person p where p.person_parent_uid=:personParentUid", nativeQuery=true)
+	List<Object[]> getPersonIdsByPersonParentId(@Param("personParentUid") Long personParentUid);
```

Comment:
Since `Person.person_uid` is a Long, you can return `List<Long>` here instead of `List<Object[]>`

Reply:
Hi @mpeels  it is  returning  a list of Long it is just that the nativeQuery will return it as a list of Object[] array with each object array containing a row for results of the specific fields specified.  I initially had it as a List<Long> which threw some errors.

Reply:
@mpeels  updated. Thanks.